### PR TITLE
Update main.c

### PR DESCRIPTION
**Pull request by AlisonLuan**
Just a small test

### CODE REVIEW

**Comment by AlisonLuan** [comment]
Hello. Please ensure that is working.

File: src/main.c [hunk 1]
```diff
@@ -43,8 +43,6 @@ typedef StaticTimer_t osStaticTimerDef_t;
 /* Private macro -------------------------------------------------------------*/
 /* USER CODE BEGIN PM */
 
-/* USER CODE END PM */
```
Comment (dagnoletto) [inline src/main.c:46]: It is ok.

**Review by dagnoletto — APPROVED** [review]
Change is approved.

**Comment by dagnoletto** [comment]
You can merge the branch back to main.

**Pull request merged**